arise

```diff
@@ -10,14 +14,27 @@ arise_version="1.1.0"
 # Don't edit below this line unless you know what you're doing
 ##############################################################
 
-cd "$(dirname $0)"
+cd "$SCRIPT_DIR"
```

Comment:
Set the default as an option so that a user can change the SCRIPT_DIR if they want to above; otherwise they leave it unset and it uses the default.

```suggestion
# Default to the script's directory if SCRIPT_DIR is not set
# https://stackoverflow.com/questions/59895/how-do-i-get-the-directory-where-a-bash-script-is-located-from-within-the-script
SCRIPT_DIR_DEFAULT=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )

# Check if SCRIPT_DIR is set and non-empty using parameter expansion; otherwise use SCRIPT_DIR_DEFAULT
cd "${SCRIPT_DIR:-$SCRIPT_DIR_DEFAULT}" || {
    echo "Error: Unable to change directory to '${SCRIPT_DIR:-$SCRIPT_DIR_DEFAULT}'"
    exit 1
}
```